### ISSUE TITLE
bug: sillent install fails to update .exe when app is open

### ISSUE THREAD

**Issue by marcinkubica** [notepad-plus-plus/notepad-plus-plus#10520]
### Description of the Issue
When running installer with /S the main binary isn't replaced if the user has the application open.
There's no error exit code emited
"Add remove" programs is reporting new version of Notepad has been installed.

### Steps to Reproduce the Issue

1. Have a version of 7.9.5 installed
2. Run notepad++
3. Run silent upgrade to ie. 8.1.4

### Expected Behavior
App upgraded

### Actual Behavior
App not upgraded
False positive in Add-Remove programs

### Debug Information
```
get-command "C:\Program Files\Notepad++\notepad++.exe"

CommandType     Name                                               Version    Source
-----------     ----                                               -------    ------
Application     notepad++.exe                                      7.9.5.0    C:\Program Files\Notepad++\notepad++.exe

```
```
Get-ItemProperty HKLM:\Software\Microsoft\Windows\CurrentVersion\Uninstall\* |  Where-Object -Property DisplayName -Like "Notepad++*" |Select-Object DisplayName, DisplayVersion, Publisher

DisplayName            DisplayVersion Publisher
-----------            -------------- ---------
Notepad++ (64-bit x64) 8.1.4          Notepad++ Team
```

### Suggestion
Non zero exit code from the installer if app binary can't be replaced
Force kill the open app

**Comment by CennoxX** [comment]
This is fixed by #14251

**Comment by donho** [comment]
https://github.com/notepad-plus-plus/notepad-plus-plus/commit/83f1195bc1f8d84080a4a243019f155a2f45f430
